dashboard/src/components/DeploymentForm/DeploymentForm.tsx

```diff
@@ -88,8 +91,9 @@ class DeploymentForm extends React.Component<IDeploymentFormProps, IDeploymentFo
     if (hr) {
       namespace = hr.metadata.namespace;
       this.setState({
+        hrName: hr.metadata.name,
         namespace,
-        releaseName: hr.metadata.name,
+        releaseName: `${hr.metadata.namespace}-${hr.metadata.name}`,
```

Comment:
when the hr object exists, the releaseName should be read from `hr.spec.releaseName`.

Reply:
i see that you've fixed it in https://github.com/kubeapps/kubeapps/pull/330